.github/workflows/interchaintest-e2e.yml

```
@@ -2,12 +2,20 @@ name: ictest E2E
 
 on:
   pull_request:
+    branches:
+      - main
+      - master
+    branches-ignore:
```

actionlint (1.7.7)

both "branches" and "branches-ignore" filters cannot be used for the same event "pull_request". note: use '!' to negate patterns

(events)